### DISCUSSION TITLE
Viem and Parcel  v2

### DISCUSSION THREAD

**Discussion by ktdasher1** [wevm/viem#517] [General]
Is anyone using Viem with parcel v2 ... it appears that parcel can't resolve viem's subfolers:

import { goerli, mainnet } from "viem/chains";
console.log(goerli, mainnet)

The above example  to force viem in seems to cause parcel not to resolve "viem/chains".  Any thoughts?

**Comment by jxom** [comment]
Ah yes, I don't believe Parcel supports `package.json#exports` yet... I think it's coming in the next major version release. We were thinking of moving everything (inc. chains) to the `viem` entrypoint as you should still be able to benefit from tree-shaking. Will look into it!

**Comment by ktdasher1** [comment]
Thanks!  I would love to use over @wagmi and ethers, but not willing to give up parcel v2

**Reply by jxom** [reply]
Just published a fix on `0.3.31` for Parcel support!
